docs/README.md

```diff
@@ -0,0 +1,42 @@
+# Gravity Docs
+
+Gravity docs are built using [mkdocs](http://www.mkdocs.org/) and hosted
+as a bunch of static files in S3. Located Here https://gravitational.com/gravity/docs/overview/ 
+
+Look at `build.sh` script to see how it works.
+
+## To Publish New Version
+
+* Update [build.sh](build.sh).
+* Update theme/base.html to add a new version to the Javascript arrray of versions
+* Create a new YAML file, like `5.5.1.yaml` if you are releasing version 5.5.1
+
+## Deploying
+
+Gravity docs are published using a private `web` repository.
+See `web/README.md` for more info.
+
+## Running Locally
+
+We recommend using Docker to run and build the docs. 
+
+`make run-docs` will run the docs and setup a [livereload server](https://chrome.google.com/webstore/detail/livereload/jnihajbhpnppcggbcgedagnkighmdlei?hl=en) for easy previewing of changes.  
```

Comment:
What does `will run the docs` mean - will start a local server to view the documentation?
I also failed to find the `run-docs` target in Makefile (there's just `run`).

Reply:
Thanks for the review @a-palchikov. 

I've been running it from the root directory but maybe I should add the command to the /docs/ directory.

Reply:
Changed `make run-docs` to `make run` for clarity.